src/test/java/org/broadinstitute/consent/http/service/EmailServiceTest.java

```diff
@@ -80,14 +96,27 @@ private void initService() {
         FreeMarkerConfiguration fmConfig = new FreeMarkerConfiguration();
         fmConfig.setDefaultEncoding("UTF-8");
         fmConfig.setTemplateDirectory("/freemarker");
-        FreeMarkerTemplateHelper helper = new FreeMarkerTemplateHelper(fmConfig);
+        templateHelper = spy(new FreeMarkerTemplateHelper(fmConfig));
         service = new EmailService(collectionDAO, consentDAO, voteDAO, electionDAO, userDAO,
-                emailDAO, sendGridAPI, helper, serverUrl);
+                emailDAO, sendGridAPI, templateHelper, serverUrl);
+    }
+
+    private void initFakeService() {
+        String serverUrl =  "http://localhost:8000/#/";
+
+        openMocks(this);
+
+        FreeMarkerConfiguration fmConfig = new FreeMarkerConfiguration();
+        fmConfig.setDefaultEncoding("UTF-8");
+        fmConfig.setTemplateDirectory("/freemarker");
+        templateHelper = spy(new FreeMarkerTemplateHelper(fmConfig));
+        service = new EmailService(collectionDAO, consentDAO, voteDAO, electionDAO, userDAO,
+                emailDAO, sendGridAPI, templateHelper, serverUrl);
     }
```

Comment:
Given that we need two different set ups to test these cases, I think this might be enough to move these tests to a separate test class.

Reply:
Turns out i was being silly and I didn't need two different setups, so this should be all good now.